js/newtab.js

```diff
@@ -203,7 +203,7 @@ function getImage(info) {
 			}
 		};
 		// Actually open the request.
-		mediaReq.open('GET', 'https://www.thebluealliance.com/api/v3/team/frc' + teamInfo.num + '/media/' + new Date().getFullYear() + '?X-TBA-Auth-Key=IJ7ECNmOibpHt04EdVs4xS7q5OQkIY5GE7USErbLXK3i4obXAilhJD8VP590o8Ur');
+		mediaReq.open('GET', 'https://www.thebluealliance.com/api/v3/team/frc' + teamInfo.num + '/media/' + (new Date().getFullYear() - 1) + '?X-TBA-Auth-Key=IJ7ECNmOibpHt04EdVs4xS7q5OQkIY5GE7USErbLXK3i4obXAilhJD8VP590o8Ur');
```

Comment:
This is the line we'll need to change, but I don't like that we're waiting until a full year has passed to show the previous year's photos. Teams typically have (at least a few of) their photos up by April or so. I'd advocate for getting the previous year's photos if the current month is less than April, and this year's if not.

Note that it will be more efficient to move the `new Date()` a line up so as to not create an extra `Date` object without need.

If you can find an efficient way to get the date 4 months previously and get the year from that object, that would be even better, but I can't figure out a way to do that while still doing fewer operations than the aforementioned system.

Reply:
Even with it at 2017, only about 1/3rd of the teams had actual photos that the extension could set as the background.

Reply:
A majority of teams will never post any photos. If they haven't posted by April or May, it's unlikely they ever will that year.